umh-core/pkg/cpuhealth/snapshot_test.go

```
@@ -0,0 +1,213 @@
+		Expect(smp.NrThrottled).To(Equal(diagnosis.Known(2)))
+		Expect(smp.Quota).To(Equal(diagnosis.Known(2)))
+		Expect(smp.Pressure).To(Equal(diagnosis.Known(0.02)))
+
+		// Spec 3: DeriveEnvironment reads exactly three facts and calls
+		// NewEnvironment. A snapshot with Virtualized true and Quota at
+		// Known(2) satisfies both capabilities; one with neither satisfies
+		// neither; and the load-bearing third case — Quota at Known(0), a
+		// PRESENT uncapped read — must NOT satisfy HasLimit (positivity, not
+		// presence). The fourth case: PsiAvailable true satisfies
+		// HasPressureStats, and its absence (the zero value) does not.
+		both := diagnosis.Environment{}
```

golangci-lint (2.12.2)

[error] ineffectual assignment to both

(ineffassign)